### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by lhorie**

### CODE REVIEW

File: docs/guides/configuring-plugins.md [hunk 1]
```diff
@@ -1,43 +0,0 @@
-# Configuring plugins
```
Comment (dennisgl) [inline docs/guides/configuring-plugins.md]: Are we planning on addressing configurations later?

File: docs/guides/dependencies.md [hunk 2]
```diff
@@ -46,33 +45,33 @@ export default () => {
 
 The `Name` plugin simply saves the value in `?name=[value]` to a cookie session if that querystring value is defined.
 
-Notice that the factory function of the `Name` plugin receives `{Session}` as an argument. This is the same `{Session}` that we passed to `app.plugin(Name, {Session})` and it's [how FusionJS plugins do dependency injection](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/configuring-plugins.md).
+Notice that the `middleware` method of the `Name` plugin receives `{Session}` as an argument. This is the same `{Session}` that we passed to `app.register(SessionToken, Session)` and it's [how FusionJS plugins do dependency injection](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/configuring-plugins.md).
```
Comment (dennisgl) [inline docs/guides/dependencies.md]: `configuring-plugins.md` is removed in this PR

File: docs/guides/getting-started.md [hunk 3]
```diff
@@ -181,22 +187,22 @@ Here are some more in-depth sections covering various aspects of FusionJS:
...
-  - [Dependencies](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/dependencies.md)
-  - [Configuring plugins](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/configuring-plugins.md)
-  - [Creating endpoints](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/creating-endpoints.md)
-  - [Creating providers](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/creating-providers.md)
-  - [Modifying the HTML template](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/modifying-html-template.md)
-  - [Working with secrets](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/working-with-secrets.md)
+* [Universal code](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/universal-code.md)
+* [Creating a plugin](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/creating-a-plugin.md)
+  * [Dependencies](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/dependencies.md)
+  * [Configuring plugins](https://github.com/fusionjs/fusion-core/blob/master/docs/guides/configuring-plugins.md)
```
Comment (dennisgl) [inline docs/guides/getting-started.md]: `configuring-plugins.md` is removed in this PR

**Comment by lhorie** [comment]
!merge

**Pull request merged**